src/pages/operator/StartStopDaemon.tsx

```diff
@@ -0,0 +1,234 @@
+      display: "flex",
+      alignItems: "center"
+    },
+    wrapper: {
+      margin: theme.spacing.unit,
+      position: "relative"
+    },
+    buttonStop: {
+      fontSize: "1.4rem",
+      backgroundColor: red[500],
+      "&:hover": {
+        backgroundColor: red[700]
+      }
+    },
+    buttonStart: {
+      fontSize: "1.4rem",
+      backgroundColor: green[500],
+      "&:hover": {
+        backgroundColor: green[700]
+      }
+    },
+    fabProgress: {
+      fontSize: "1.4rem",
+      color: green[500],
+      position: "absolute",
+      top: -6,
+      left: -6,
+      zIndex: 1
+    },
+    buttonProgress: {
+      fontSize: "1.4rem",
+      color: green[500],
+      position: "absolute",
+      top: "50%",
+      left: "50%",
+      marginTop: -12,
+      marginLeft: -12
+    },
+    snackbarSuccess: {
+      fontSize: "1.8rem",
+      backgroundColor: green[600]
+    },
+    snackbarError: {
+      fontSize: "1.8rem",
+      backgroundColor: theme.palette.error.dark
+    }
+  });
+
+interface IProps {
+  classes: any;
+  network: any;
+  daemonEndpoint: string;
+}
+
+interface IState {
+  loading: boolean;
+  daemonRunning: boolean;
+  showAlert: boolean;
+  alertText: string;
+  alertVariant: string;
+}
+
+class StartStopDaemon extends Component<IProps, IState> {
+  public userAddress: string = "";
+
+  public state = {
+    loading: false,
+    daemonRunning: true,
+    showAlert: false,
+    alertText: "",
+    alertVariant: "success"
+  };
+
+  private handleButtonClick = async (): Promise<any> => {
+    this.setState({ loading: true });
+    const { network, daemonEndpoint } = this.props;
+    const { daemonRunning } = this.state;
+    let commandRequest: CommandRequest = new CommandRequest();
+
+    if (this.userAddress === "") {
+      this.userAddress = await network.getAccount();
+    }
+    let currentBlockNumber: number = await network.getCurrentBlockNumber();
+    let command = daemonRunning ? 1 : 0; // 1 - Stop Request, 0 - Start Request
+
+    commandRequest.setUserAddress(this.userAddress);
+    commandRequest.setCurrentBlock(currentBlockNumber);
+    commandRequest.setCommand(command);
+
+    let msg: string = network.composeSHA3Message(
+      ["string", "uint256", "address"],
+      ["_Request_Read", currentBlockNumber, this.userAddress]
+    );
+
+    network.eth
+      .personal_sign(msg, this.userAddress)
+      .then((signed: any): void => {
+        let signature = network.buffSignature(signed);
+        commandRequest.setSignature(signature);
+        try {
+          grpc.invoke(ConfigurationService.StopProcessingRequests, {
+            request: commandRequest,
+            host: daemonEndpoint,
+            onMessage: (): void => {},
+            onEnd: (code: Code, msg: string | undefined): void => {
+              if (code === 0 && daemonRunning) {
+                this.setState({
+                  loading: false,
+                  daemonRunning: false,
+                  showAlert: true,
```

Comment:
Lets have all messages in its own file so there is only one place to modify it and also support translations if required in future